### PR TITLE
Merge in feature/annotations

### PR DESCRIPTION
**Pull request by brianmwadime**

### CODE REVIEW

File: Example/Pods/Target Support Files/UXMPDFKit/Info.plist [hunk 1]
```diff
@@ -2,25 +2,25 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-  <key>CFBundleDevelopmentRegion</key>
```
Comment (diegostamigni) [inline Example/Pods/Target Support Files/UXMPDFKit/Info.plist:5]: Please revert all the changes in this file.

File: Example/UXMPDFKit.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -164,7 +164,7 @@
 				TargetAttributes = {
 					607FACCF1AFB9204008FA782 = {
 						CreatedOnToolsVersion = 6.3.1;
-						DevelopmentTeam = 9592M4L4J8;
```
Comment (diegostamigni) [inline Example/UXMPDFKit.xcodeproj/project.pbxproj:167]: Same here, please revert all the changes in this file.

File: Example/UXMPDFKit/ExampleViewController.swift [hunk 3]
```diff
@@ -18,9 +18,10 @@ class ExampleViewController: UIViewController {
         
         let pdf = UXMPDFViewController(document: document!)
         pdf.annotationController.annotationTypes = [
-            PDFHighlighterAnnotation.self,
-            PDFPenAnnotation.self,
-            UXMTextAnnotation.self
+//            PDFHighlighterAnnotation.self,
```
Comment (diegostamigni) [inline Example/UXMPDFKit/ExampleViewController.swift:21]: Get rid of this comments if you don't need them anymore.

File: Pod/Classes/Form/UXMFormPageView.swift [hunk 4]
```diff
@@ -129,3 +136,25 @@ open class UXMFormPageView: UIView {
...
+  }
+
+  public func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldReceive touch: UITouch) -> Bool {
+    if (activeWidgetAnnotationView == nil) { return false }
+    let rect = activeWidgetAnnotationView?.frame
+    if(!(rect?.contains(touch.location(in: self)))!) {
+      if (activeWidgetAnnotationView?.isKind(of: UITextView.self))! {
+        activeWidgetAnnotationView?.resignFirstResponder()
+      } else {
+//        [_activeWidgetAnnotationView resign];
```
Comment (diegostamigni) [inline Pod/Classes/Form/UXMFormPageView.swift:153]: Get rid of this whole else block

File: Pod/Classes/Annotations/UXMSignAnnotation.swift [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+  }
+}
+
+extension UXMSignAnnotation: UXMPDFAnnotationButtonable {
+
+  public static var name: String? { return "Sign" }
+  public static var buttonImage: UIImage? { return UIImage.bundledImage("sign") }
+}
+
+//extension UXMSignAnnotation: UITextViewDelegate {
```
Comment (diegostamigni) [inline Pod/Classes/Annotations/UXMSignAnnotation.swift:112]: If not used, please get rid of it

File: Pod/Classes/Annotations/UXMSignAnnotation.swift [hunk 6]
```diff
@@ -0,0 +1,225 @@
...
+  override required public init() { super.init() }
+
+  public func didEnd() {
+  }
+
+  required public init(coder aDecoder: NSCoder) {
+    page = aDecoder.decodeObject(forKey: "page") as? Int
+    image = aDecoder.decodeObject(forKey: "image") as? UIImage
+    rect = aDecoder.decodeCGRect(forKey: "rect")
+//    font = aDecoder.decodeObject(forKey: "font") as! UIFont
```
Comment (diegostamigni) [inline Pod/Classes/Annotations/UXMSignAnnotation.swift:43]: Get rid of this here and below

**Review by diegostamigni — CHANGES_REQUESTED** [review]
Hi, thanks for this PR! I've left some minor comments to it. Please have a look.

**Comment by brianmwadime** [comment]
Not yet done with this. Was meant to merge in my forks master.

**Pull request closed without merging**